src/99-appendices/07-meg-systems.md

```diff
@@ -27,4 +27,4 @@ Restricted keywords for ManufacturersModelName field in the `*_meg.json` file:
 | KIT-160               | KIT/Yokogawa          |                                                                                              |
 | KIT-208               | KIT/Yokogawa          |                                                                                              |
 | ITAB-ARGOS153         | ITAB                  |                                                                                              |
-| Aalto-MEG-MRI-YYYY/MM | Aalto/MEG–MRI         | YYYY-MM (year, month; or major version)                                                      |
+| Aalto-MEG-MRI-YYYY/MM | Aalto/MEG/MRI         | YYYY-MM (year, month; or major version)                                                      |
```

Comment:
same here: I think we cannot change this restricted keyword

Reply:
I'd made this change because for some reason LaTeX to pdf was  errori-ng out with hyphens. I don't recall why that was happening but I've reverted it back to `Aalto/MEG–MRI` on line 30 and line 11. Tested the pdf build and it's working fine. Will make a commit once I address the remaining issues.